orchagent/pfcwdorch.cpp

```diff
@@ -488,7 +491,7 @@ bool PfcWdSwOrch<DropHandler, ForwardHandler>::registerInWdDb(const Port& port,
     set<uint8_t> losslessTc;
     for (uint8_t i = 0; i < PFC_WD_TC_MAX; i++)
     {
-        if ((pfcMask & (1 << i)) == 0)
+        if ((pfcRxMask & (1 << i)) == 0)
```

Comment:
How do you handle the case where pfcwd is already started with asym pfc disabled for a port and later asym pfc is enabled? In that case, for the pfcwd to act on all priorities, we need to restart it.

Reply:
Yes, we need to restart it. Original plan was to leave it under user responsibility. So once PFC configuration is changed then PFCWD should be restarted in order to apply the changes for PFCWD. BTW, I am not sure if it is a good idea to restart PFCWD automatically because it might affect some traffic flows. And user knows when to restart PFCWD in order to avoid or minimize traffic impact. Also usually both PFCWD and Asymmetric PFC are configured once in JSON config file and in such case all should be working without any restarts.

Reply:
I agree it is not a good idea to restart pfcwd automatically. but we do have a cli option ' pfc config asymmetric' to enable/disable asymmetric pfc. So we need to add in some comments so that user is aware that pfcwd will see these changes only after a restart